docs/quick-starts/feature-flag-quickstart.md

```diff
@@ -17,27 +17,57 @@ Give the flag a descriptive human readable name and create a single variation fo
 
 ![Feature gate 1](/img/feature-flagging/feature-gate-1.png)
 
-### Create allocations to describe your target audience
+### 2. Create allocations to describe your target audience
 
 After creating the flag, decide who to target by creating [allocations](/feature-flags#allocations). In this case we will create two allocations that describe our target audience for the new page: internal users and half of all North American web users:
 
 ![Feature gate 2](/img/feature-flagging/feature-gate-2.png)
 
 Create each allocation one by one, giving each a name and specifying the traffic split and traffic exposure. Finally, enter rules that specify which subjects are part of that allocation. In the example above `Internals users` are determined by their email, and `North America Web Users` are determined by their country and device. For this allocation we'll set the traffic exposure to 50% so that 50% of the group still sees the original page (SDK will return `null`).
 
-### Initialize the SDK
+### 3. Connect a logging function to the Eppo SDK (optional)
+
+If you are using feature flags to implement a randomized experiment, you will need to log each time a user is assigned a variant. Instead of integrating an additional event logging system, Eppo connects to your existing data warehouse logging infrastructure. Whether you are using a third party system to log events to the data warehouse or have an internally built solution, you'll simply pass in a logging function when initializing the SDK. 
```

Comment:
We should mention in the intro that this quickstart assumes you have some system to log. It would suck to get to this part and realize you are stuck.

Reply:
Agreed. I'll make a note to add that as a fast follow